src/migrations/user-store/5.0.0-alpha.2.ts

```diff
@@ -0,0 +1,18 @@
+// Switch representation of hiddenTableColumns in store
+import { migration } from "../migration-wrapper";
+
+export default migration({
+  version: "5.0.0-alpha.2",
```

Comment:
We should bump this version, right?

Reply:
This really shouldn't be here at all, will delete these two files.